server/sonar-server/src/main/java/org/sonar/server/issue/ws/SearchAction.java

```diff
@@ -362,7 +366,7 @@ private void completeFacets(Facets facets, SearchWsRequest request, Request wsRe
     if (actionPlansFromRequest != null) {
       actionPlans.addAll(actionPlansFromRequest);
     }
-    addMandatoryValuesToFacet(facets, ACTION_PLANS, actionPlans);
+    addMandatoryValuesToFacet(facets, DEPRECATED_ACTION_PLANS, actionPlans);
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png) Remove this use of "DEPRECATED_ACTION_PLANS"; it is deprecated. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://dory.sonarsource.com/coding_rules#rule_key=squid%3ACallToDeprecatedMethod)